test/TestApps/PushNotificationsDemoPackage/PushNotificationsDemoPackage.wapproj

```diff
@@ -48,6 +48,7 @@
     <GenerateTestArtifacts>True</GenerateTestArtifacts>
     <AppxBundlePlatforms>$(Platform)</AppxBundlePlatforms>
     <HoursBetweenUpdateChecks>0</HoursBetweenUpdateChecks>
+    <AppxPackageDir>C:\Users\vemancha\Desktop\</AppxPackageDir>
```

Comment:
Make sure you cleanup local changes 😄

Reply:
This should be a relative path I imagine.

Reply:
@pmpurifoy that is why still in draft mode :)